### PR TITLE
[Enhancement] Add some debug info for txn publish process

### LINKED ISSUE #9950: Add debug info for db lock acquiring and txn publish process

### PR DESCRIPTION
**Pull request by nshangyiming**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #9950 

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by nshangyiming** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 39 / 47 (82.98%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/transaction/PublishVersionDaemon.java|0|2|00.00%|[177, 205]|
|:large_blue_circle:|com/starrocks/catalog/Database.java|16|22|72.73%|[134, 135, 136, 148, 150, 153]|
|:large_blue_circle:|com/starrocks/common/Config.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/transaction/DatabaseTransactionMgr.java|4|4|100.00%|[]|
|:large_blue_circle:|com/starrocks/leader/LeaderImpl.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/transaction/TransactionState.java|11|11|100.00%|[]|
|:large_blue_circle:|com/starrocks/task/PublishVersionTask.java|3|3|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/leader/LeaderImpl.java [hunk 1]
```diff
@@ -590,6 +591,10 @@ private void finishPublishVersion(AgentTask task, TFinishTaskRequest request) {
             }
         }
         publishVersionTask.setIsFinished(true);
+        TransactionState txnState = publishVersionTask.getTxnState();
+        if (txnState != null) {
+            txnState.updatePublishTaskFinishTime();
```
Comment (decster) [inline fe/fe-core/src/main/java/com/starrocks/leader/LeaderImpl.java:596]: This is only valid if all tasks are finished normally and within Config.quorom_publish_wait_time_ms
Do we have a better way to make it more accurate?

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java [hunk 2]
```diff
@@ -117,6 +117,13 @@ public class Config extends ConfigBase {
     @ConfField
     public static String audit_log_delete_age = "30d";
 
+    @ConfField(mutable = true)
+    public static long slow_lock_threshold_ms = 3000L;
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/common/Config.java:121]: The two configuration can be unified to one configuration.

Comment (nshangyiming) [reply fe/fe-core/src/main/java/com/starrocks/common/Config.java:121]: Although currently they have the same value, but they have different usages, for example, when i was doing some tests, i want to set `slow_lock_log_every_ms` to a relative low value, say `100ms`, but keep `slow_lock_threshold_ms` unchanged.

File: fe/fe-core/src/main/java/com/starrocks/leader/LeaderImpl.java (hunk 1 above)
Comment (nshangyiming) [reply fe/fe-core/src/main/java/com/starrocks/leader/LeaderImpl.java:596]: Yes, this is only to give a rough judge of the publish rpc cost of the txn, more accurate statistics like rpc cost of different be can be done, but i think currently it's not that necessary, so i will keep this logic unchanged for now.

**Review by decster — APPROVED** [review]

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**